src/app.ts

```diff
@@ -25,6 +27,23 @@ import { FHIRStructureDefinitionRegistry } from './registry';
 
 const configVersionSupported: ConfigVersion = 1;
 
+function prepareRequestContext(req: express.Request): RequestContext {
```

Comment:
Should this function be here? I didn't find a good helper TS file. I don't want to put it in the interface package because it depends on express and interface doesn't seem to depend on anything.

Reply:
hmm yeah I think you are right -- a util.ts file would be ideal but I don't seen an issue add the message here